misk/src/test/kotlin/misk/web/jetty/WebActionsServletTest.kt

```diff
@@ -0,0 +1,131 @@
+package misk.web.jetty
+
+import com.netflix.concurrency.limits.limit.SettableLimit
+import com.netflix.concurrency.limits.limiter.SimpleLimiter
+import misk.Action
+import misk.ApplicationInterceptor
+import misk.asAction
+import misk.client.UnixDomainSocketFactory
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.time.FakeClock
+import misk.time.FakeClockModule
+import misk.web.Get
+import misk.web.NetworkChain
+import misk.web.NetworkInterceptor
+import misk.web.ResponseContentType
+import misk.web.SourceSocket
+import misk.web.WebActionModule
+import misk.web.WebTestingModule
+import misk.web.WebUnixDomainSocketConfig
+import misk.web.actions.LivenessCheckAction
+import misk.web.actions.ReadinessCheckAction
+import misk.web.actions.StatusAction
+import misk.web.actions.WebAction
+import misk.web.interceptors.UserInterceptorTest
+import misk.web.mediatype.MediaTypes
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Protocol
+import okhttp3.Request
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.time.Duration
+import java.util.UUID
+import javax.inject.Inject
+
+@MiskTest(startService = true)
+class WebActionsServletTest {
+  @MiskTestModule
+  val module = TestModule()
+
+  @Inject
+  internal lateinit var jettyService: JettyService
+
+  private var socketName: String = "@udstest" + UUID.randomUUID().toString()
+
+  @Test
+  fun networkSocketSuccess() {
+    val response = get("/potato", SourceSocket.Type.NETWORK)
+    assertThat(response.header("ActualSocketType")).isEqualTo("NETWORK")
+    assertThat(response.header("ActualSocketName")).isEqualTo(
+      jettyService.httpServerUrl.toString().removeSurrounding("http://", "/"))
```

Comment:
`"${jettyService.httpServerUrl.host}:${jettyService.httpServerUrl.port}"`

Reply:
that's much better, forgot httpServerUrl wasn't a string